tests/models/sam/test_processor_sam.py

```diff
@@ -55,12 +57,8 @@ def tearDown(self):
         shutil.rmtree(self.tmpdirname)
 
     def prepare_image_inputs(self):
-        """This function prepares a list of PIL images, or a list of numpy arrays if one specifies numpify=True,
-        or a list of PyTorch tensors if one specifies torchify=True.
-        """
-        image_inputs = [np.random.randint(255, size=(3, 30, 400), dtype=np.uint8)]
-        image_inputs = [Image.fromarray(np.moveaxis(x, 0, -1)) for x in image_inputs]
-        return image_inputs
+        """This function prepares a list of PIL images."""
+        return prepare_image_inputs()
```

Comment:
I guess this is similar to mgp-str where we can't use `ProcessorTesterMixin`, a comment explaining that would be nice

Reply:
I've added comments - LMK if they're OK